docs/Course Wiki/CS Course Offerings/CS426.md

```diff
@@ -0,0 +1,74 @@
+# CS426
+
+CS426 (Compiler Construction) is a 3 (undergrad) or 4 (grad) credit hour course that fulfills an Advanced Computing requirement for Computer Engineering majors. It is offered only in the fall semester.
+
+## Content Covered
+This course teachs both the theory behind compilers and optimizations and practical ways to implement them.
```

Comment:
"teaches" is misspelled.  change to "optimizations, as well as practical ways to implement them."